distribution/packages/build.gradle

```diff
@@ -42,10 +42,18 @@ import java.util.regex.Pattern
  *    dpkg -c path/to/elasticsearch.deb
  */
 
-plugins {
-  id "nebula.ospackage-base" version "8.3.0"
+buildscript {
+  repositories {
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+  }
+  dependencies {
+    classpath "com.github.breskeby:gradle-ospackage-plugin:ddb72a9922b934033827d48d296f7f3d470ac422"
```

Comment:
Why do we need to fork this?

Reply:
I'm waiting for a release of the ospackage plugin that includes my PR that fixed a stackoverflow within the plugin: https://github.com/nebula-plugins/gradle-ospackage-plugin/pull/397